samples/EchoBot/MyService.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Samples.Echo
+{
+    public interface IMyService
+    {
+        Task DoSomethingAsync();
+    }
+
+    public sealed class MyService : IMyService
+    {
+        public Task DoSomethingAsync() => Task.Delay(500);
```

Comment:
async. Wouldn't want to leak exceptions. :) 

(I learned that from you!)

Reply:
Well, since I'm returning the `Task` to the caller, the caller upstream will observe the exception directly from that `Task`, so no need for async/await here.